imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -79,6 +79,7 @@
 ]
 
 
+@pytest.mark.download_test_data()
```

Comment:
I think you might be able to put this up at the module level if every test needs this data:
https://docs.pytest.org/en/stable/example/markers.html

```
pytestmark = pytest.mark.external_test_data
```